gsoc/proposal-Data-Visualization-For-Archived-Content.md

```diff
@@ -0,0 +1,97 @@
+## Google Summer of Code - Application Template
+
+### Student
+
+**Name:** Harsh Baid
+
+**Affiliation:** Student - Computer Science & double minor in Astronomy/Math
+
+**Location:** University of Maryland - College Park
+
+**GitHub handle:** @Blackglade
+
+**Slack handle:** @Blackglade
+
+**Email:** hbaid@umd.edu OR harsh@harshbaid.com
+
+**Other Contact:**  http://harshbaid.com/ (website)
+
+### Project Details
+
+**Project Title:**  
+Data Visualization of Archived Content
+
+**Project Abstract/Summary (<20 words):**  
+The purpose of this project is to develop a set of visually impactful and meaningful models that allow users to easily visualize the changes archived by EDGI.
+
+**Describe the need your project fills:**  
+The Environmental Data and Governance Initiative (EDGI) challenges an administration who have made it a proactive effort to shut down or dilute key federal agencies responsible for the protection of the environment. You guys do this by working with a network of individuals to proactively archive and monitor all public environmental data so that it will be available in the future to the public.
+
+This data, while vital for the community as a whole, is useless in the hands of the average individual. To put it in simple terms, the average laymen just doesn’t have the time or the enormous effort required to sit through each changing variable and try to decipher which of it is relatively “meaningful.” Most individuals looking to do research/reports in these areas, although happy to discover the archived data, will be dissuaded due to the enormous task at hand of sifting through each minor change. 
+
+**Describe how your project meets this need:**  
+This is where my project comes in. Using data visualization libraries (such as https://d3js.org/), my project’s goal will be to develop a set of models/graphs accurately displaying any “meaningful” data you have archived. I would ideally be using the DataRefuge API for this project as a means to gather data for my in-browser model, but could eventually directly integrate with your toolkit to make API calls. These visualizations could be added directly on top of your React.js app view layer (https://www.archivers.space/). Working with my assigned mentors to identify how this data is “relevant” and then constructing an appropriate model to display it would allow individuals visiting your app to gain a better understanding of the data being collected. This would be a preferential solution, as opposed to sifting through tons of lists.
+
+**Milestones/Timeline:**  
+This project is essentially composed of **3 major milestones**: working with my mentors to identify which data is meaningful and worth developing a model for, developing the model while simultaneously receiving feedback on improvements, and then eventually committing it to production for use. Below you will find a rough timeline:

```

Comment:
These milestones seem great, the only change I'd advise is we'd ask ask you to commit continuously to production, at first with a minimum viable version of an initial visualization, and then continuously improve it over time before moving on to the next. Your third milestone could instead be documenting what you learned for the greater community to read about.